client/tests/webdriver/baseSpecs/showOrganization.spec.js

```diff
@@ -61,6 +62,22 @@ describe("Show organization page", () => {
       )
       expect(assignedTasksShortNames).to.have.members(EF2_ASSIGNED_TASKS)
     })
+    it("Should see sync matrix on the Show page", async() => {
+      const syncMatrix = await ShowOrganization.getSyncMatrix()
+      await syncMatrix.waitForExist()
+      await syncMatrix.waitForDisplayed()
+
+      expect(await (await syncMatrix.$(".legend")).getText()).to.contain(
+        "Sync matrix for EF 2"
```

Comment:
I suggest to extend this test, e.g. to check that for EF 1, all (sub)tasks for EF 1 are shown, and that for EF 1.1, only the (sub)tasks for EF 1.1 are shown.